cmd/mt-index-migrate/main.go

```diff
@@ -76,30 +78,31 @@ func main() {
 
 	// ensure the dest table exists.
 	schemaTable := util.ReadEntry(*schemaFile, "schema_table").(string)
-	err = dstSession.Query(fmt.Sprintf(schemaTable, *dstKeyspace)).Exec()
+	err = dstSession.Query(fmt.Sprintf(schemaTable, *dstKeyspace, *dstIdxTable)).Exec()
 	if err != nil {
 		log.Fatalf("cassandra-idx failed to initialize cassandra table. %s", err.Error())
 	}
 
 	wg.Add(1)
-	go writeDefs(dstSession, defsChan)
+	go writeDefs(dstSession, defsChan, *dstIdxTable)
 	wg.Add(1)
-	go getDefs(srcSession, defsChan)
+	go getDefs(srcSession, defsChan, *srcIdxTable)
 
 	wg.Wait()
 
 }
 
-func writeDefs(session *gocql.Session, defsChan chan *schema.MetricDefinition) {
+func writeDefs(session *gocql.Session, defsChan chan *schema.MetricDefinition, idxTable string) {
 	log.Info("starting write thread")
 	defer wg.Done()
 	counter := 0
 	pre := time.Now()
 	for def := range defsChan {
-		qry := `INSERT INTO metric_idx (id, orgid, partition, name, interval, unit, mtype, tags, lastupdate) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)`
+		qry := fmt.Sprintf("INSERT INTO %s (id, orgid, partition, name, interval, unit, mtype, tags, lastupdate) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)", idxTable)
```

Comment:
We can't use the cql replacement syntax here ?

Reply:
I can't remember any specific reason. It should work. I would have to do that in the following places for consistency:

Reply:
I'm not sure if that's the case for cassandra as well, but I know that in MySQL it's usually recommended to use the client lib's variable replacement mechanism because it does the correct character escaping for you. So that might be a good idea.

Reply:
param replacement doesnt work for keyspace and table names.  The fmt-sprintf approach is needed.